packages/ag-charts-enterprise/src/features/sync/chartSync.ts

```diff
@@ -52,15 +53,11 @@ export class ChartSync extends BaseProperties implements _ModuleSupport.ModuleIn
         super();
     }
 
-    private updateChart(chart: ChartLike, updateType = ChartUpdateType.UPDATE_DATA) {
-        chart.ctx.updateService.update(updateType, { skipSync: true });
-    }
-
-    private updateSiblings(groupId?: string) {
+    updateSiblings(groupId?: string) {
         const { syncManager } = this.moduleContext;
 
         const updateFn = async () => {
-            for (const chart of syncManager.getGroupSiblings(groupId)) {
+            for (const chart of syncManager.getGroupSiblings(groupId ?? this.groupId)) {
                 await chart.waitForDataProcess(120);
```

Comment:
Q: Does this need to be done serially, or should we be making these calls in (pseudo-)parallel?

Reply:
We could. You're not worried we may put too much strain on the engine, blocking UI on a large scale of synced charts?